theme/n21/views/help/partials/help_form_invite_external_members.hbs

```diff
@@ -6,5 +6,7 @@
 
 <h2 class="h3 mt-2">Wen möchtest du ins Team einladen?</h2>
 <button type="button" data-role="teacher" class="btn btn-set-role" >Lehrer anderer Schulen</button>
-<button disabled type="button" data-role="expert" class="btn btn-set-role">Externe Experten</button>
-<p class="hint">{{$t "teams._team.members.n21.text.inviteToTeamHint"}}</p>
+<button type="button" data-role="expert" class="btn btn-set-role">Externe Experten</button>
+
+{{!-- "Externe Experten" feature activated. To deactivate uncomment the line (below) and disabled the button (above)  --}}
+{{!-- <p class="hint">{{$t "teams._team.members.n21.text.inviteToTeamHint"}}</p> --}}
```

Comment:
I would remove this and the I18n string, since it specifically refers to the date of the deactivation, so it must be changed anyway the next time (and I feel like there will never be a next time)